src/persist-client/examples/open_loop.rs

```diff
@@ -71,7 +71,7 @@ pub struct Args {
     record_size_bytes: usize,
 
     /// Batch size in number of records (if applicable).
-    #[clap(long, env = "", value_name = "R", default_value_t = 10)]
+    #[clap(long, env = "", value_name = "R", default_value_t = 100)]
```

Comment:
changed this because the defaults before were a batch every 0.1s which seemed fast to me for a throughput benchmark